Makefile

```diff
@@ -21,7 +21,9 @@ IMAGE_PREFIX                := $(REGISTRY)/extensions
 REPO_ROOT                   := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 HACK_DIR                    := $(REPO_ROOT)/hack
 VERSION                     := $(shell cat "$(REPO_ROOT)/VERSION")
-LD_FLAGS                    := $(shell ./vendor/github.com/gardener/gardener/hack/get-build-ld-flags.sh github.com/gardener/"$(EXTENSION_PREFIX)-$(NAME)/pkg" "$(REPO_ROOT)/VERSION")
+LD_FLAGS_GENERATOR          := $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/get-build-ld-flags.sh
+LD_FLAGS                    := $(shell $(LD_FLAGS_GENERATOR) k8s.io/component-base $(REPO_ROOT)/VERSION $(EXTENSION_PREFIX))
```

Comment:
Can you amend the following change?

```diff
diff --git a/.dockerignore b/.dockerignore
index 7599eb2..b11bf06 100644
--- a/.dockerignore
+++ b/.dockerignore
@@ -2,6 +2,7 @@
 **

 # Exclude folders relevant for build
+!.git
 !charts/
 !cmd/
 !docs/
```

Otherwise the `GitCommit` field is not populated:
```
$ docker run eu.gcr.io/gardener-project/gardener/extensions/runtime-gvisor:v0.5.0-dev-7f4c8361bdad633351aa44b320a6fb53b22af825 --version=raw
version.Info{Major:"0", Minor:"5+", GitVersion:"v0.5.0-dev", GitCommit:"", GitTreeState:"clean", BuildDate:"2022-02-28T14:05:54+00:00", GoVersion:"go1.17.5", Compiler:"gc", Platform:"linux/amd64"}
```

Reply:
Done.